dist/amazon-linux/2/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM amazonlinux:2 AS build
+
+ENV REFRESHED_AT 20220926T210646Z
+ENV LANG=C.UTF-8
+ENV DEBIAN_FRONTEND=noninteractive
+ENV MIX_ENV=prod
+
+# Set the right versions
+ENV ERLANG_VERSION 25.1.1
+ENV ELIXIR_VERSION 1.14.1
+ENV ASDF_DATA_DIR /opt/asdf
+ENV ASDF_DIR /opt/asdf
```

Comment:
This can be removed
```suggestion
```

Reply:
I was trying to use asdf before but realized that compiling from source is easier